Packs/SignalSciences/Integrations/SignalSciences/SignalSciences.py

```diff
@@ -1335,7 +1335,7 @@ def get_sites_command():
 
 def fetch_incidents():
     now_utc = datetime.utcnow()
-    most_recent_event_time = None
+    most_recent_event_time = ''
```

Comment:
How does it solve the issue?
Isn't `event_time` on line 1361 of type DateTime?

Reply:
`datetime_to_posix_without_milliseconds` converts it to str